docs/source/notebooks/wps_CI_demo.ipynb

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
<p>Thank you for including an <code>assert</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>